src/js/SortSelect/SortSelect.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import { styled } from '@material-ui/core/styles';
+
+import { bool, func } from 'prop-types';
+import headersMapping from '../api/tableHeadersMapping';
+
+const NasaForm = styled(FormControl)({
+  minWidth: 80,
+});
+
+class SortSelect extends Component {
+  handleChange = event => {
+    this.props.sortData(this.props.isDescendingSort, event.target.value);
+  };
+
+  render() {
+    return (
+      <NasaForm className="form">
+        <InputLabel id="nasa-simple-select-label">Sort by:</InputLabel>
+        <Select
+          labelId="nasa-simple-select-label"
+          id="nasa-simple-select"
+          autoWidth={true}
+          onChange={this.handleChange}
+        >
+          {Object.keys(headersMapping).map(element => {
+            if (element === 'Full camera name') return true;
+            return (
+              <MenuItem key={element} value={element}>
+                {element}
+              </MenuItem>
+            );
+          })}
```

Comment:
It looks too complicated. Maybe you turn it to ternary? And why do you `true` in case you don't need to return anything? Usually we return _null_ in that cases because React doesn't render it